src/common/blob-provider.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export class BlobProvider {
```

Comment:
Just curious if this can just be an exported function rather than a class with a static function.